diff --git a/OpenDF/OpenDF-UI/app/components/Button.js b/OpenDF/OpenDF-UI/app/components/Button.js
index e69de29..606e199 100644
--- a/OpenDF/OpenDF-UI/app/components/Button.js
+++ b/OpenDF/OpenDF-UI/app/components/Button.js
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default class Button extends React.Component {
+
+  render() {
+    return (
+      <button className="button button-primary">
+        <i className="fa fa-chevron-right"></i> Find out more
+      </button>
+    )
+  }
+}
diff --git a/OpenDF/OpenDF-UI/app/components/Card.js b/OpenDF/OpenDF-UI/app/components/Card.js
index e69de29..5b9099e 100644
--- a/OpenDF/OpenDF-UI/app/components/Card.js
+++ b/OpenDF/OpenDF-UI/app/components/Card.js
@@ -0,0 +1,12 @@
+import React from 'react';
+import CardBody from './CardBody';
+
+export class Card extends React.Component {
+  render() {
+    return (
+      <article className="card">
+        <CardBody title={this.props.details.title} text={this.props.details.text}/>
+      </article>
+    )
+  }
+}
diff --git a/OpenDF/OpenDF-UI/app/components/CardBody.js b/OpenDF/OpenDF-UI/app/components/CardBody.js
index e69de29..681e4af 100644
--- a/OpenDF/OpenDF-UI/app/components/CardBody.js
+++ b/OpenDF/OpenDF-UI/app/components/CardBody.js
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from './Button';
+
+export default class CardBody extends React.Component {
+
+  render() {
+    return (
+      <div className="card-body">
+
+        <h2>{this.props.title}</h2>
+
+        <p className="body-content">{this.props.text}</p>
+
+        <Button />
+      </div>
+    )
+  }
+}
diff --git a/OpenDF/OpenDF-UI/app/containers/HomePage/index.js b/OpenDF/OpenDF-UI/app/containers/HomePage/index.js
index 3fa0549..8517914 100644
--- a/OpenDF/OpenDF-UI/app/containers/HomePage/index.js
+++ b/OpenDF/OpenDF-UI/app/containers/HomePage/index.js
@@ -12,13 +12,36 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import messages from './messages';
+import {Card} from '../../components/Card';
+import PostsData from '../../data.js'

 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+  constructor() {
+    super();
+
+    this.state = {
+      posts: {}
+    }
+  }
+  componentWillMount() {
+    this.setState({
+      posts: PostsData
+    });
+  }
+
+
   render() {
     return (
-      <h1>
-        <FormattedMessage {...messages.header} />
-      </h1>
-    );
+      <div>
+          <header className="app-header"></header>
+          <div className="app-card-list" id="app-card-list">
+            {
+              Object
+              .keys(this.state.posts)
+              .map(key => <Card key={key} index={key} details={this.state.posts[key]}/>)
+            }
+        </div>
+      </div>
+    )
   }
 }
diff --git a/OpenDF/OpenDF-UI/app/containers/Profile/avatar.js b/OpenDF/OpenDF-UI/app/containers/Profile/avatar.js
index e69de29..67215f5 100644
--- a/OpenDF/OpenDF-UI/app/containers/Profile/avatar.js
+++ b/OpenDF/OpenDF-UI/app/containers/Profile/avatar.js
@@ -0,0 +1,32 @@
+/*
+ * Profile Messages
+ *
+ * This contains all the text for the Profile component.
+ */
+// import { defineMessages } from 'react-intl';
+import React from 'react';
+
+export class Avatar extends React.Component {
+  render() {
+    var image = this.props.image,
+        style = {
+          width: this.props.width || 50,
+          height: this.props.height || 50
+        };
+
+    if (!image) return null;
+
+    return (
+     <div className="avatar" style={style}>
+           <img src={this.props.image} />
+      </div>
+    );
+  }
+}
+
+// export default defineMessages({
+//   header: {
+//     id: 'app.containers.Profile.header',
+//     defaultMessage: 'This is Profile Your File !',
+//   },
+// });
diff --git a/OpenDF/OpenDF-UI/app/containers/Profile/index.js b/OpenDF/OpenDF-UI/app/containers/Profile/index.js
index 57fce3b..089c0b5 100644
--- a/OpenDF/OpenDF-UI/app/containers/Profile/index.js
+++ b/OpenDF/OpenDF-UI/app/containers/Profile/index.js
@@ -8,7 +8,21 @@ import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import Helmet from 'react-helmet';
 import { FormattedMessage } from 'react-intl';
-import messages from './messages';
+import styled from 'styled-components';
+import {MainPanel} from './panel';
+
+
+//Dummy data. These should be pulled from the API
+var user = {
+  basicInfo: {
+    name: "Jane Doe",
+    gender: "Female",
+    birthday: "April 3, 1990",
+    location: "Los Angeles, CA",
+    photo: "http://lorempixel.com/500/500/people",
+    bio: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat fugit quia pariatur est saepe necessitatibus, quibusdam reiciendis ratione voluptate atque in qui provident rem repellat soluta. Blanditiis repellat velit eligendi."
+  }
+}

 export class Profile extends React.Component { // eslint-disable-line react/prefer-stateless-function
   render() {
@@ -20,7 +34,9 @@ export class Profile extends React.Component { // eslint-disable-line react/pref
             { name: 'description', content: 'Description of Profile' },
           ]}
         />
-        <FormattedMessage {...messages.header} />
+        <div id="user-profile">
+          <MainPanel info={user.basicInfo} ></MainPanel>
+        </div>
       </div>
     );
   }
diff --git a/OpenDF/OpenDF-UI/app/containers/Profile/panel.js b/OpenDF/OpenDF-UI/app/containers/Profile/panel.js
index e69de29..3afc98b 100644
--- a/OpenDF/OpenDF-UI/app/containers/Profile/panel.js
+++ b/OpenDF/OpenDF-UI/app/containers/Profile/panel.js
@@ -0,0 +1,45 @@
+/*
+ * Profile Messages
+ *
+ * This contains all the text for the Profile component.
+ */
+// import { defineMessages } from 'react-intl';
+import React from 'react';
+import {Avatar} from './avatar';
+
+
+export class MainPanel extends React.Component {
+  render() {
+    var info = this.props.info;
+    if (!info) return null;
+
+    return (
+     <div>
+        <div className="top">
+            <Avatar
+               image={info.photo}
+               width={100}
+               height={100}
+            />
+            <h2>{info.name}</h2>
+            <h3>{info.location}</h3>
+
+          <hr />
+            <p>{info.gender} | {info.birthday}</p>
+        </div>
+
+        <div className="bottom">
+          <h4>Biography</h4>
+          <p>{info.bio}</p>
+        </div>
+      </div>
+    );
+  }
+}
+
+// export default defineMessages({
+//   header: {
+//     id: 'app.containers.Profile.header',
+//     defaultMessage: 'This is Profile Your File !',
+//   },
+// });
diff --git a/OpenDF/OpenDF-UI/app/data.js b/OpenDF/OpenDF-UI/app/data.js
index e69de29..d524e6b 100644
--- a/OpenDF/OpenDF-UI/app/data.js
+++ b/OpenDF/OpenDF-UI/app/data.js
@@ -0,0 +1,34 @@
+const PostsData = [
+  {
+    "title": "Pasan Ranathunga",
+    "text": "CNN purchased Casey Neistat's Beme app for $25million.",
+    "image": "https://source.unsplash.com/user/erondu/600x400"
+  },
+  {
+    "title": "Malith Senaweera",
+    "text": "Learn our tips and tricks on living a nomadic lifestyle",
+    "image": "https://source.unsplash.com/user/_vickyreyes/600x400"
+  },
+  {
+    "title": "Pasan Ranathunga",
+    "text": "The first ever decoupled starter theme for React & the WP-API",
+    "image": "https://source.unsplash.com/user/ilyapavlov/600x400"
+  },
+  {
+    "title": "rehrumesh",
+    "text": "CNN purchased Casey Neistat's Beme app for $25million.",
+    "image": "https://source.unsplash.com/user/erondu/600x400"
+  },
+  {
+    "title": "Pasan Ranathunga",
+    "text": "Learn our tips and tricks on living a nomadic lifestyle",
+    "image": "https://source.unsplash.com/user/_vickyreyes/600x400"
+  },
+  {
+    "title": "Agent Milindu",
+    "text": "The first ever decoupled starter theme for React & the WP-API",
+    "image": "https://source.unsplash.com/user/ilyapavlov/600x400"
+  }
+]
+
+export default PostsData;
diff --git a/OpenDF/OpenDF-UI/app/global-styles.js b/OpenDF/OpenDF-UI/app/global-styles.js
index df5326f..43a719b 100644
--- a/OpenDF/OpenDF-UI/app/global-styles.js
+++ b/OpenDF/OpenDF-UI/app/global-styles.js
@@ -27,4 +27,101 @@ injectGlobal`
     font-family: Georgia, Times, 'Times New Roman', serif;
     line-height: 1.5em;
   }
+
+  #user-profile {
+    width: 100%;
+    max-width: 450px;
+    margin: 70px auto;
+    border-radius: 5px;
+    text-align: center;
+    overflow: hidden;
+    box-shadow: 2px 2px 5px #000;
+    border: 1px solid #000;
+    @include dotBg(darken($green, 5%), $green, 30px);
+
+    hr {
+        margin: 20px auto;
+        width: 50px;
+        border: 1px solid $magenta;
+        background-color: transparent;
+      }
+
+    h1, h2, h3, h4, h5, h6, p {
+        margin: 0;
+      }
+
+      h2 {
+        font-size: 30px;
+      }
+
+      h3 {
+        font-size: 18px;
+      }
+
+      h4 {
+        font-size: 16px;
+
+      }
+
+      p {
+        font-size: 14px;
+      }
+
+      .top, .bottom {
+        padding: 30px;
+
+      }
+
+      .bottom {
+
+        background-color: rgba(white, 0.3);
+        text-align: left;
+      }
+
+      .avatar {
+        border-radius: 50%;
+        overflow: hidden;
+        display: inline-flex;
+        margin-bottom: 15px;
+        border: 1px solid #000;
+        img {
+          width: 100%;
+        }
+      }
+  }
+
+  .app-card-list {
+    display:inline-block;
+    width:100%;
+    float:right;
+    margin:0;
+    min-height: 75vh;
+    white-space:normal;
+    overflow-x: scroll;
+    margin-top:-70px;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+
+    @media(min-width:768px) {
+        margin-top: 0;
+    }
+
+    .card {
+      margin: 60px 30px;
+      display: inline-block;
+      box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+      .card-body {
+          white-space: normal;
+          padding: 5%;
+      }
+    }
+  }
+
+  .designer-link {
+    display:inline-block;
+    padding-top:40px;
+  }
 `;
